Services/Notifications/classes/Provider/NotificationMainBarProvider.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace ILIAS\Notifiactions\Provider;
+
+use ILIAS\GlobalScreen\Helper\BasicAccessCheckClosures;
+use ILIAS\GlobalScreen\Scope\MainMenu\Provider\AbstractStaticMainMenuProvider;
+use ILIAS\MainMenu\Provider\StandardTopItemsProvider;
+use ilObjNotificationAdmin;
+
+/**
+ * Class NotificationMainBarProvider
+ * @author Ingmar Szmais <iszmais@databay.de>
+ *
+ */
+class NotificationMainBarProvider extends AbstractStaticMainMenuProvider
```

Comment:
Can you please elaborate what this provider is for? The notification administration should be par of the `Tree`, so why do we need an explicit provider?

Reply:
As far as i know the Menü ist collected by all providers wich implement StaticMainMenuProvider so this seems neccessary to appear inside the menu.
Pls Correct me if im wrong

Reply:
Okay, the reason is the nature and position of the main bar entry. It is similar to the `OrganisationalUnit` entry, so a dedicated `MainBar Provider` has to be created: https://docu.ilias.de/goto_docu_wiki_wpage_6494_1357.html